docs/content/en/docs/overview/_index.md

```diff
@@ -30,7 +30,7 @@ Component Architecture
 - Automated deployment analysis to measure deployment impact based on metrics, logs, emitted requests
 - Automatically roll back to the previous state as soon as analysis or a pipeline stage fails
 - Automatically detect configuration drift to notify and render the changes
-- Automatically watch and detect the new container images to deploy
+- Automatically update configuration files when it finds out a new event
```

Comment:
nit
`Automatically trigger a new deployment when a defined event has occurred (eg. helm chart updated, container image updated, etc)`

Reply:
Lol. I had the same comment at the same time.

Reply:
interesting 🤣

Reply:
You guys suggested almost the same one lol Thank you, let me use a kind of that.